crates/iota-storage/src/object_store/mod.rs

```diff
@@ -33,24 +33,32 @@ macro_rules! as_ref_get_ext_impl {
 as_ref_get_ext_impl!(Arc<dyn ObjectStoreGetExt>);
 as_ref_get_ext_impl!(Box<dyn ObjectStoreGetExt>);
 
-#[async_trait]
-impl ObjectStoreGetExt for Arc<DynObjectStore> {
-    async fn get_bytes(&self, src: &Path) -> Result<Bytes> {
-        self.get(src)
-            .await
-            .map_err(|e| anyhow!("Failed to get file {} with error: {:?}", src, e))?
-            .bytes()
-            .await
-            .map_err(|e| {
-                anyhow!(
-                    "Failed to collect GET result for file {} into bytes with error: {:?}",
-                    src,
-                    e
-                )
-            })
-    }
+macro_rules! as_ref_get_impl {
+    ($type:ty) => {
+        #[async_trait]
+        impl ObjectStoreGetExt for $type {
+            async fn get_bytes(&self, src: &Path) -> Result<Bytes> {
+                println!("getting bytes");
```

Comment:
```suggestion
```
you added this line for debugging? it's non existent on upstream at least. please remove.